src/pages/workspace/expensifyCard/issueNew/ConfirmationStep.tsx

```diff
@@ -58,7 +58,12 @@ function ConfirmationStep({policyID, backTo}: ConfirmationStepProps) {
         if (!isSuccessful) {
             return;
         }
-        Navigation.goBack(backTo ?? ROUTES.WORKSPACE_EXPENSIFY_CARD.getRoute(policyID));
+        if (backTo) {
+            Navigation.goBack(backTo);
+        } else {
+            Navigation.navigate(ROUTES.WORKSPACE_EXPENSIFY_CARD.getRoute(policyID));
+        }
```

Comment:
This logic is missing from `src/pages/workspace/companyCards/assignCard/ConfirmationStep.tsx`